utils/envvars/parser.go

```diff
@@ -135,6 +145,37 @@ func parseStringDuration(value string) (time.Duration, error) {
 	return d, nil
 }
 
+// GetSAASHost validates SAAS host env variables.
+// returns it if it's valid, otherwise returns defaultSAASHost.
+func GetSAASHost() string {
+	switch {
+	case govalidator.IsURL(os.Getenv(envSAASAuthHost)):
```

Comment:
If `PERCONA_TEST_AUTH_HOST` contains not-url, then it is error, and user should be notified. Probably with app crash.